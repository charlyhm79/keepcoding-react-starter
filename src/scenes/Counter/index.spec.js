import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Counter from './';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Counter />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('Matchs snapshot', () => {
  const tree = renderer.create(<Counter />).toJSON();
  expect(tree).toMatchSnapshot();
});
