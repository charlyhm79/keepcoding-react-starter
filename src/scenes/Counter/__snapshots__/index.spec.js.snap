// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matchs snapshot 1`] = `
<div
  className="sc-ifAKCX haxXVA sc-bxivhb fYKwXF"
>
  <p
    className="sc-EHOje iMmpZp"
  >
    El contador está vacío
  </p>
  <div
    className="sc-htpNat cixthU"
  >
    <button
      className="sc-bdVaJa bqVjqO"
      disabled={true}
      onClick={[Function]}
      to={undefined}
    >
      Reset
    </button>
    <button
      className="sc-bdVaJa hpXSGV"
      onClick={[Function]}
      to={undefined}
    >
      Add!
    </button>
  </div>
</div>
`;
